content/fedlex/orders/clear-policy-promise.md

```diff
@@ -5,10 +5,10 @@ aliases= ["/order/0001", "/fedlex/O0001"]
 +++
 
 ## I. What does this policy effect
-All documents produced by The Farer Group including, but not limited to: policies, orders, and dociuments necessary for a member to act on their rights; must be written in plain language. These documents should be written in English and have translations into French and Spanish, where phesibly possible.
+All documents produced by The Farer Group including, but not limited to: policies, orders, and dociuments necessary for a member to act on their rights; must be written in plain language. These documents should be written in English and have translations into French, where phesibly possible.
```

Comment:
Can you fix the typo for "policies, orders, and dociuments" ("dociuments" should be "documents") while you're at it please?